modules/phase_field/test/tests/grain_tracker_test/distributed_poly_ic.i

```diff
@@ -0,0 +1,271 @@
+  [voronoi_id]
+  []
+  [evaluable_elems]
+  []
+[]
+
+[Kernels]
+  # Kernel block, where the kernels defining the residual equations are set up.
+  [PolycrystalKernel]
+    # Custom action creating all necessary kernels for grain growth.  All input parameters are up in GlobalParams
+  []
+[]
+
+[AuxKernels]
+  # AuxKernel block, defining the equations used to calculate the auxvars
+  [bnds_aux]
+    # AuxKernel that calculates the GB term
+    type = BndsCalcAux
+    variable = bnds
+    execute_on = 'initial timestep_end'
+  []
+  [unique_grains]
+    type = FeatureFloodCountAux
+    variable = unique_grains
+    flood_counter = grain_tracker
+    field_display = UNIQUE_REGION
+    execute_on = 'initial timestep_end'
+  []
+  [var_indices]
+    type = FeatureFloodCountAux
+    variable = var_indices
+    flood_counter = grain_tracker
+    field_display = VARIABLE_COLORING
+    execute_on = 'initial timestep_end'
+  []
+  [ghosted_entities]
+    type = FeatureFloodCountAux
+    variable = ghost_regions
+    flood_counter = grain_tracker
+    field_display = GHOSTED_ENTITIES
+    execute_on = 'initial timestep_end'
+  []
+  [halos]
+    type = FeatureFloodCountAux
+    variable = halos
+    flood_counter = grain_tracker
+    field_display = HALOS
+    execute_on = 'initial timestep_end'
+  []
+  [halo0]
+    type = FeatureFloodCountAux
+    variable = halo0
+    map_index = 0
+    field_display = HALOS
+    flood_counter = grain_tracker
+  []
+  [halo1]
+    type = FeatureFloodCountAux
+    variable = halo1
+    map_index = 1
+    field_display = HALOS
+    flood_counter = grain_tracker
+  []
+  [halo2]
+    type = FeatureFloodCountAux
+    variable = halo2
+    map_index = 2
+    field_display = HALOS
+    flood_counter = grain_tracker
+  []
+  [halo3]
+    type = FeatureFloodCountAux
+    variable = halo3
+    map_index = 3
+    field_display = HALOS
+    flood_counter = grain_tracker
+  []
+  [halo4]
+    type = FeatureFloodCountAux
+    variable = halo4
+    map_index = 4
+    field_display = HALOS
+    flood_counter = grain_tracker
+  []
+  [halo5]
+    type = FeatureFloodCountAux
+    variable = halo5
+    map_index = 5
+    field_display = HALOS
+    flood_counter = grain_tracker
+  []
+  [halo6]
+    type = FeatureFloodCountAux
+    variable = halo6
+    map_index = 6
+    field_display = HALOS
+    flood_counter = grain_tracker
+  []
+  [halo7]
+    type = FeatureFloodCountAux
+    variable = halo7
+    map_index = 7
+    field_display = HALOS
+    flood_counter = grain_tracker
+  []
+  [centroids]
+    type = FeatureFloodCountAux
+    variable = centroids
+    execute_on = 'timestep_end'
+    field_display = CENTROID
+    flood_counter = grain_tracker
+  []
+  [proc_id]
+    type = ProcessorIDAux
+    variable = proc_id
+    execute_on = 'initial'
+  []
+  [voronoi_id]
+    type = VoronoiICAux
+    variable = voronoi_id
+    execute_on = 'initial'
+    polycrystal_ic_uo = voronoi
+  []
+[]
+
+[BCs]
+  # Boundary Condition block
+[]
+
```

Comment:
What boundary conditions are we using here? `Neumann`? We may not need this block.

Reply:
No flux - this is completely fine for GrainGrowth type simulations.